plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HiveMetastoreModule.java

```diff
@@ -45,6 +46,7 @@ protected void setup(Binder binder)
             bindMetastoreModule("thrift", new ThriftMetastoreModule());
             bindMetastoreModule("file", new FileMetastoreModule());
             bindMetastoreModule("glue", new GlueMetastoreModule());
+            bindMetastoreModule("dlf", new DlfMetaStoreModule());
```

Comment:
to an outsider person like me, "DLF" is an acronym that has little meaning. Can we name it more descriptively?

This applies to metastore type, all the classes, and config name prefix.

Reply:
Any suggestions? DLF has no alias up to now, maybe AlibabaDlf? Or the full name DataLakeFormation? Which one is better? The latter seems too long...

Reply:
`AlibabaDlf` for class names, `alibaba-dlf` for metastore config value and ``alibaba-dlf.` as config prefix works for me, thanks!

Reply:
The config prefix cannot be changed because of API limitations, others refactored already.

Reply:
> The config prefix cannot be changed because of API limitations

i guess it's because the config names are inherited, right?
commented here: https://github.com/trinodb/trino/pull/11239/files#r825781313